mysql-test/mysql-test-run.pl

```diff
@@ -5984,9 +5989,9 @@ ($)
   vardir=DIR            The directory where files generated from the test run
                         is stored (default: ./var). Specifying a ramdisk or
                         tmpfs will speed up tests.
-  mem                   Run testsuite in "memory" using tmpfs or ramdisk
-                        Attempts to find a suitable location
-                        using a builtin list of standard locations
+  mem[=DIR]             Run testsuite in "memory" using tmpfs or ramdisk
+                        Attempts to use DIR first if specified else
+                        uses as builtin list of standard locations
```

Comment:
"Attempts to use DIR first if specified else uses as builtin list" -- it should be "else uses **a** builtin list", right?